### DISCUSSION TITLE
Overlay Doesn’t Work for 4K 16:9 24fps for HERO11

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#539] [Q&A] [answered]
I recently tried this Labs feature:
[Overlays](https://gopro.github.io/labs/control/overlays/)

and got the date & time formatting as I wanted it. According to the text it should work on the HERO11:
> HERO10/11 support is limited to video modes 4K 16:9 up to 30, 2.7K up to 60 and 1080p up to 120 fps.

I tried recording in 4K 16:9 at 24 fps with the 24 fps corresponding to 24 Hz option enabled.
This did not work at all; there was no overlay of any kind on the resulting video.

However when I set the resolution to 1080p 16:9 it worked perfectly.

Am I misreading or misunderstanding the quoted text?

**Comment by dnewman-gpsw** [comment]
You also need to be in 8-bit, then 4Kp24 work fine.

**Reply by AlexLandherr** [reply]
Ok, understood.

Though when I set the QR code generator to disable overlays it doesn't seem to work as I still see a date and time overlay at the top of the image.

**Reply by dnewman-gpsw** [reply]
Explain.  What are you do to display (vs powering off)?

**Reply by AlexLandherr** [reply]
10-bit mode is off; I try to film in 4K 16:9 24 fps (24 Hz) and I see something like this at the center top of the video frame:
```
    12:02:47 U
    2023-09-16
```

I suspect that's a remnant of when I earlier today experimented with the size, positioning and format of the date and time overlay.

**Reply by dnewman-gpsw** [reply, marked as answer]
So you used a permanent command.  Best to practice without enabling the permanent option. Removing permanent requires you do the same command set to zero (for numbers) or "" for strings.  Or just !RESET to erase everything you may have set with Labs.

**Comment by AlexLandherr** [comment]
> So you used a permanent command. Best to practice without enabling the permanent option. Removing permanent requires you do the same command set to zero (for numbers) or "" for strings. Or just !RESET to erase everything you may have set with Labs.

I used this URL to get the !RESET command; now when I reboot the camera it doesn't list any enabled features which I understand means I've removed any previous Labs settings
[https://gopro.github.io/labs/control/set/?cmd=!RESET](https://gopro.github.io/labs/control/set/?cmd=!RESET)

**Reply by dnewman-gpsw** [reply]
Correct, reset removes them all.
